ios/ReactTestApp/Info.plist

```diff
@@ -65,8 +65,8 @@
 			</array>
 		</dict>
 	</dict>
-	<key>UILaunchStoryboardName</key>
-	<string>LaunchScreen</string>
+	<key>UILaunchScreen</key>
```

Comment:
Turns out this requires iOS 14+ so I retargeted to the 3.0 branch. I will also have to bump iOS minimum version.